src/actions/pkg.cr

```diff
@@ -1,44 +1,146 @@
 module DppmRestApi::Actions::Pkg
   extend self
+  ALL_PKGS = ""
+  ONE_PKG  = "/:id"
+
+  def output_config_for(package, to builder)
+    builder.field package.package do
+      builder.object do
+        package.pkg_file.config_vars.try &.each_key do |key|
+          builder.field key, package.get_config key
+        end
+      end
+    end
+  end
+
+  def find_package_by_name(name, pfx) : DPPM::Prefix::Pkg?
+    DPPM::Prefix.new(pfx).each_pkg { |pkg| return pkg if pkg.package == name }
```

Comment:
Instantiate directly the package, then checking if it exists.
Remember that a package name is `mypackage_1.0.0`.
Else, if we want info for `mypackage`, that's the source one (`Src`)